features/admin/lib/src/ui/trash_window.dart

```diff
@@ -221,9 +223,21 @@ class _TrashWindowState extends State<TrashWindow> {
                 ),
               ),
               16.heightBox,
-              Text(
-                trash.name,
+              TextFormField(
+                initialValue: name,
+                onChanged: (String value) {
+                  setState(() {
+                    name = value;
+                  });
+                },
                 style: CustomStyles.body1,
+                decoration: InputDecoration(
```

Comment:
This code isn't as DRY as it could be, as it's duplicated in other sections, like the `aad_answer_tab`. To enhance maintainability and avoid repetition, consider consolidating this common logic into a reusable component e.g. `AdminTextFormField`. This way, we adhere to the DRY principle, promoting code consistency and reducing redundancy.